### PR TITLE
[WFLY-11764] Analysis for WFLY-11764

### PR DESCRIPTION
**Pull request by fl4via**
WFLY Jira: https://issues.jboss.org/browse/WFLY-11764
EAP7 Jira: https://issues.jboss.org/browse/EAP7-1341

### CODE REVIEW

File: undertow/WFLY-11764_UndertowOptions_PRESERVE_PATH_ON_FORWARD.doc [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+* mailto:bspyrkos@redhat.com[Bartosz Spyrko-Smietanko]
+
+=== QE Contacts
+
+* mailto:jstourac@redhat.com[Jan Stourac]
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[ ] Engineering
```
Comment (jstourac) [inline undertow/WFLY-11764_UndertowOptions_PRESERVE_PATH_ON_FORWARD.doc]: please, select engineering here :)

File: undertow/WFLY-11764_UndertowOptions_PRESERVE_PATH_ON_FORWARD.doc [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+* WildFly, Undertow component
+* Undertow
+
+=== Other Interested Projects
+
+
+== Requirements
+
+This will be implemented as an additional attribute under the `subsystem=undertow` management model address.
```
Comment (jstourac) [inline undertow/WFLY-11764_UndertowOptions_PRESERVE_PATH_ON_FORWARD.doc]: actually under the `/subsystem=undertow/servlet-container=<name>`?

Comment (fl4via) [reply undertow/WFLY-11764_UndertowOptions_PRESERVE_PATH_ON_FORWARD.doc]: @jstourac  Good point, I'll update

File: undertow/WFLY-11764_UndertowOptions_PRESERVE_PATH_ON_FORWARD.doc (hunk 1 above)
Comment (fl4via) [reply undertow/WFLY-11764_UndertowOptions_PRESERVE_PATH_ON_FORWARD.doc]: @jstourac I wanted to double check with you before marking that, I'll update with the requested change :-)

**Comment by fl4via** [comment]
@jstourac can you please review again? When trying to trace the Undertow release under which this has been added, I noticed that UndertowOptions was moved to DeploymentConfig.presevePathOnForward under UNDERTOW-1504... there are so many Jiras for this feature that even I lost track :)

**Comment by jstourac** [comment]
@fl4via, thank you for the update. I have no more comments on this, LGTM :)

You are right that UNDERTOW-1504 introduced possibility to configure the feature via a system property. Although, actual behaviour change has been made already by this issue fix https://issues.jboss.org/browse/UNDERTOW-1499 which says it made it into 2.0.19.Final.

**Pull request merged**